lib/components/admin/field-trip-details.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+
+import * as callTakerActions from '../../actions/call-taker'
+import DraggableWindow from './draggable-window'
+import Icon from '../narrative/icon'
+
+/**
+ * Shows the details for the active Field Trip Request.
+ */
+class FieldTripDetails extends Component {
+  _onCloseActiveFieldTrip = () => {
+    this.props.setActiveFieldTrip(null)
+  }
+
+  render () {
+    const {callTaker, request} = this.props
+    if (!request) return null
+    const {id, schoolName} = request
+    const {fieldTrip} = callTaker
+    const defaultPosition = {...fieldTrip.position}
+    defaultPosition.x = defaultPosition.x - 460
+    return (
+      <DraggableWindow
+        draggableProps={{ defaultPosition }}
+        header={
+          <h4>
+            <Icon type='graduation-cap' /> {schoolName} Trip (#{id})
+          </h4>
+        }
+        onClickClose={this._onCloseActiveFieldTrip}
+        style={{width: '450px'}}
+      >
+        {Object.keys(request).map(key => {
+          const value = typeof request[key] === 'object'
+            ? 'TODO: object'
```

Comment:
What is this?

Reply:
This needs to be implemented.

<img width="387" alt="Screen Shot 2021-01-15 at 4 50 14 PM" src="https://user-images.githubusercontent.com/3112493/104792026-c7252580-5751-11eb-8c40-8399a2000ead.png">

Reply:
PR has been updated to render full details.